app/src/main/java/de/xikolo/models/Item.java

```diff
@@ -157,6 +183,10 @@ public Item convertToRealmObject() {
             item.visited = visited;
             item.accessible = accessible;
 
+            if (timeEffort != null) {
+                item.timeEffort = timeEffort;
+            }
+
```

Comment:
As the `Item.timeEffort` is `null` by default and nullable, this null-check is redundant.

Reply:
The realm field is a primitive `int`, the JSON one a nullable wrapper `Integer`. That's because the API doesn't expose this key for some platforms when this feature is not enabled. On the app side, we're fine with a `0` default value, in which case we don't display the value.

The check is required since you cannot assign a `null` to a primitive `int`.

Reply:
Alright, makes sense. Then we should add the default value.

Reply:
It's Java, the default value for `int` is always `0` even without an explicit assignment.